ClientApp/Podfile

```diff
@@ -0,0 +1,46 @@
+source 'https://github.com/firebase/SpecsDev.git'
```

Comment:
@paulb777 have we delete this file? Probably include here because of some rebase conflict I had before.